src/components/util/util.test.js

```diff
@@ -366,6 +380,30 @@ describe('getContributors', () => {
   });
 });
 
+describe('isConsortiumEnabled', () => {
+  it('should return false', () => {
+    const data = isConsortiumEnabled();
+    expect(data).toBeFalsy();
+  });
+
+  it('should return true', () => {
+    const data = isConsortiumEnabled(STRIPES);
+    expect(data).toBe(true);
+  });
```

Comment:
~Under what circumstances should it return true? This makes it sound like it should always return true. Describe the situation you are testing, e.g. "returns true given a valid 'centralTenantId'". OOC, what happens/should happen when you pass a `stripes` object with different kinds of sparse data, e.g.~ 
```
user.user.consortium === {}
user.user.consortium.centralTenantId === ''
user.user.consortium.centralTenantId === false
user.user.consortium.centralTenantId === null
```

Eh, never mind, this is somewhat covered in the tests below. The test descriptions could still be improved though.